### PR TITLE
[Merged by Bors] - feat: port Data.Polynomial.DenomsClearable

### PR DESCRIPTION
**Pull request by ChrisHughes24**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Polynomial/DenomsClearable.lean [hunk 1]
```diff
@@ -87,28 +87,28 @@ end DenomsClearable
 
 open RingHom
 
+set_option synthInstance.etaExperiment true in
```
Comment (RemyDegenne) [inline Mathlib/Data/Polynomial/DenomsClearable.lean:90]: Why is that needed?

**Review by jcommelin — COMMENTED** [review]
bors d+
